lib/MusicBrainz/Server/Sitemap/Overall.pm

```diff
@@ -323,7 +323,8 @@ sub construct_url_lists($$$@) {
 
         if ($suffix_info{paginated}) {
             # 100 items per page, and the first page is covered by the base.
-            my $paginated_count = ceil($id_info->{$suffix_info{paginated}} / 100) - 1;
+            my $item_count = $id_info->{ $suffix_info{paginated} } // 0;
+            my $paginated_count = ceil($item_count / 100) - 1;
```

Comment:
Oops, this commit was unrelated to the PR, but something I pushed to production earlier (and it stuck here because I rebased this PR on master, and it wasn't in master yet).  Doesn't matter though.